optin/reporting/models/database.js

```diff
@@ -0,0 +1,22 @@
+const { Sequelize } = require('sequelize');
+
+const dbList = ['completed', 'in_progress'];
+
+const database = {};
+
+const refreshDBs = () => {
+	for (let i = 0; i <= dbList.length - 1; i++) {
+		const db = dbList[i];
+		database[db] = new Sequelize(db, null, null, {
```

Comment:
can you open readonly?

Reply:
I didn't find any options for that.
I will continue dig on docs

Reply:
i think you just need to append `?mode=ro` to connection string. if it is configured correctly, write operations should fail.

Reply:
> i think you just need to append `?mode=ro` to connection string. if it is configured correctly, write operations should fail.

it does not work, after the append `?mode=ro` 
got the `SequelizeDatabaseError` when doing query

Reply:
By the way i found the solution 👍🏼